Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+/// Renders a receipt in an AirPrint enabled printer.
+/// To be properly implemented in https://github.com/woocommerce/woocommerce-ios/issues/3978
+final class ReceiptRenderer: UIPrintPageRenderer {
+    private let lines: [ReceiptLineItem]
+    private let paymentIntent: PaymentIntent
+
+    private let attributes: [NSAttributedString.Key: Any] = [.font: UIFont(name: "HelveticaNeue", size: 48) as Any]
+
+    init(content: ReceiptContent) {
+        self.lines = content.lineItems
+        self.paymentIntent = content.paymentIntent
+
+        super.init()
+
+        configureHeaderAndFooter()
+
+        configureFormatter()
+    }
+
+    override func drawHeaderForPage(at pageIndex: Int, in headerRect: CGRect) {
+        let pageNumberString = NSString(string: "Order receipt. Page \(pageIndex + 1)")
```

Comment:
Needs localization.

Continuous roll receipt printers won't have page numbers. Do we need to open an issue to handle formatting for continuous roll vs sheet paper receipts?

Reply:
Good point! I added this just to print something for now, to visualize that it actually works. I added a note to the ticket we have for iterating on the renderer (#3978), so that we remove page numbers.